pydatastructs/strings/trie.py

```diff
@@ -104,6 +104,24 @@ def is_present(self, string: str) -> bool:
             walk = walk.get_child(char)
         return True
 
+    def is_present(self, string: str) -> bool:
+        """
+        Checks if the given string is present in the trie.
+        Parameters
+        ==========
+        string: str
+        Returns
+        =======
+        True if the given string is present in trie;
+        False in all other cases.
+        """
+        walk = self.root
+        for char in string:
+            if walk.get_child(char) is None:
+                return False
+            walk = walk.get_child(char)
+        return walk.is_terminal
```

Comment:
Seems like this means the string should be fully present in the trie? If its as a prefix then it will return `False`? Well, even if a string is present as a prefix it is still there in the trie.